### ISSUE TITLE
Opening second ComfyUI window kills running one

### ISSUE THREAD

**Issue by afk4life** [comfyanonymous/ComfyUI#4851]
### Expected Behavior

Previously if I were running a workflow and wanted to open a new window to compare the workflow to a previous one, it would just keep running. Now, the newly opened window goes into "run" (tab color changes purple) and kills the running process. 

### Actual Behavior

Pretty simple -- open ComfyUI, start running a workflow, new tab, go to ComfyUI, it kills the running workflow.

### Steps to Reproduce

Pretty simple -- open ComfyUI, start running a workflow, new tab, go to ComfyUI, it kills the running workflow.

### Debug Logs

```powershell
There's nothing beyond "interrupted" in the logs to explain. One thing I've noticed is the load time of ComfyUI is significantly longer in the past few weeks because it's trying to load something in models.
```

**Comment by robinjhuang** [comment]
Which custom nodes are you using? Can't reproduce using a default workflow at least on my laptop.

**Comment by afk4life** [comment]
As_ComfyUI_CustomNodes
AuraSR-ComfyUI
cg-image-picker
comfy-image-saver
ComfyUI-AutomaticCFG
ComfyUI-BiRefNet
ComfyUI-Chibi-Nodes
ComfyUI-Crystools
ComfyUI-Custom-Scripts
ComfyUI-Eagle-PNGInfo
ComfyUI-Easy-Use
ComfyUI-Florence2
ComfyUI-Flowty-LDSR
ComfyUI-Frame-Interpol
ComfyUI-GGUF
ComfyUI-GlifNodes
ComfyUI-Image-Saver
ComfyUI-Impact-Pack
ComfyUI-Inspire-Pack
ComfyUI-Jjk-Nodes
ComfyUI-KJNodes
ComfyUI-Long-CLIP
ComfyUI-Loopchain
ComfyUI-Lora-Auto-Trig
comfyui-magic-clothing
ComfyUI-Manager
comfyui-mask-util
ComfyUI-MotionDiff
ComfyUI-MultiGPU
ComfyUI-NAI-styler
ComfyUI-ODE
comfyui-oms-diffusion
comfyui-portrait-maste
ComfyUI-post-processin
comfyui-prompt-reader-
ComfyUI-QualityOfLifeS
comfyui-reactor-node
ComfyUI-RK-Sampler
ComfyUI-SAI_API
ComfyUI-TiledDiffusion
comfyui-tooling-nodes
ComfyUI-Universal-Styl
ComfyUI-Video-Matting
ComfyUI-WD14-Tagger
ComfyUI_Comfyroll_Cust
comfyui_controlnet_aux
ComfyUI_essentials
ComfyUI_experiments
ComfyUI_FaceAnalysis
ComfyUI_FaceSimilarity
comfyui_face_parsing
ComfyUI_InstantID
ComfyUI_IPAdapter_plus
ComfyUI_LayerStyle
ComfyUI_NetDist
ComfyUi_NNLatentUpscal
ComfyUI_Noise
ComfyUI_TensorRT
ComfyUI_tinyterraNodes
ComfyUI_UltimateSDUpsc
eden_comfy_pipelines
efficiency-nodes-comfy
extended-saveimage-com
Image-Captioning-in-Co
klinter_nodes
mikey_nodes
Plush-for-ComfyUI
rgthree-comfy
save-image-extended-co
sd-perturbed-attention
sdxl_prompt_styler
sigmas_tools_and_the_g
Skimmed_CFG
stability-ComfyUI-node
was-node-suite-comfyui

**Comment by ltdrdata** [comment]
If you're talking about running one ComfyUI instance, opening two tabs in the browser, executing in one tab and stopping in the other, then that's perfectly normal.

**Comment by ltdrdata** [comment]
Is the same phenomenon occurring even when all custom nodes are disabled?

**Comment by afk4life** [comment]
I haven't disabled anything to test, but this is a recent phenomenon. What
I'm saying is if I'm running a workflow in a tab, then open a new tab and
go to 127.0.0.1... it automatically terminates the running process even
though all I am doing is going to the 127.0.0.1 site in a different window
so I can compare workflows between it and the running one

**Comment by ltdrdata** [comment]
> I haven't disabled anything to test, but this is a recent phenomenon. What I'm saying is if I'm running a workflow in a tab, then open a new tab and go to 127.0.0.1... it automatically terminates the running process even though all I am doing is going to the 127.0.0.1 site in a different window so I can compare workflows between it and the running one
> […](#)
> On Sun, 8 Sept 2024 at 21:35, Dr.Lt.Data ***@***.***> wrote: Is the same phenomenon occurring even when all custom nodes are disabled? — Reply to this email directly, view it on GitHub <http://click.dexterous.io/ls/click?upn=u001.vABqkSU068yVqUiKjBYsS4NJ9DYfLfieEvymOPoKJuOr29HKAkcBUli4F5wA0VyXEjm6qtM81GzSqcuS1HnMxsyYMLPDzFmRUiefci35blnn8aGPceC1PZNB5COEE20OzsgX_51r8jo3L-2BhAoYVmO4X6H1xXDrxq96z6T3KglwxOShBlwatOCrKAxQZqyAt9sbwkLhTwnimeiSNIzKLBggryW6iK5J7HQUukA6H0ccdC5Iw-2BmEvypqHn8VvjPDHVfwb4VEmUdD7ze1dx9PpBtSl6LlTASiQdr8Ezc7MJ-2FSk1Y1-2F5bgSrIvaqye0VEbwixrKDOp43gmxG4PC14rQ-2F6U-2FlkNM9CDQnOV2U2CiAVrWeQaPdHqNwCcNHZQK0-2BgfDYKsh-2Famkl2n1z4aO-2FkUbxbe96tw-3D-3D>, or unsubscribe <http://click.dexterous.io/ls/click?upn=u001.vABqkSU068yVqUiKjBYsS8lmWPO6P2wEPwxkZM-2FxGoHlLVsK7U9kRloKUAC5YOhHfMglzB1flZRUSnAHfCPLs-2BQiVv5RblCsnfAciM-2FMnvftSmkW21RhFoVZtSwbCB6st7B0PUAFK7WR9DyRe11q-2BMtJB0AQlEp9s5v4bdpAwCIu15Jk4gAqDUoGg-2F9-2BjiPpBGXBoZIOQKbGz3K-2B0pKwmT2Icm1xQDF8gPqg3sguqCg-3D5MbF_51r8jo3L-2BhAoYVmO4X6H1xXDrxq96z6T3KglwxOShBlwatOCrKAxQZqyAt9sbwkLhTwnimeiSNIzKLBggryW6iK5J7HQUukA6H0ccdC5Iw-2BmEvypqHn8VvjPDHVfwb4VhZiIiB8USBkKTk-2FZfAgT1T1TBWdvk4Os9aSwfB2ev093r5M90v85uzgnKHCXs5CZX98TejdOfcdnPvw5iwveaPf6wJFf-2BhqV64srPFtGfyhqqyRdaNszWwfJUA7o7BixOyFGvvnjm3ixLd-2Fc1yCyHw-3D-3D> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Are you saying that it's not just the workflow being interrupted, but that the process is actually being killed?

**Comment by afk4life** [comment]
The entire running workflow shuts down if I go to or open a second tab and go to the ComfyUI interface in it

**Comment by ltdrdata** [comment]
The interruption of a workflow's execution and the termination of a process are completely different concepts.

First of all, in your current situation, it's impossible to identify the exact cause of the issue.
Start by disabling all custom nodes and testing from there.

**Comment by afk4life** [comment]
With a suspect in mind, I went past disabling Quality of Life suite and uninstalled. Tried once, the workflow did not stop, though I am still getting

Retrying request to /models in 0.996235 seconds
Retrying request to /models in 1.768147 seconds
Retrying request to /openai/v1/models in 0.756736 seconds
Retrying request to /openai/v1/models in 1.838466 seconds

No idea what that would even be at this point. But at least whatever caused it may have been taken out of the picture.
